src/Symfony/Installer/NewCommand.php

```diff
@@ -133,7 +139,22 @@ private function checkSymfonyVersionIsInstallable()
     {
         // 'latest' is a special version name that refers to the latest stable version
         // available at the moment of installing Symfony
-        if ('latest' === $this->version) {
+        if (self::VERSION_LATEST === $this->version) {
+            return $this;
+        }
+
+        // Get the Symfony versions list (http://symfony.com/versions.json)
+        $versionsList = $this->getVersionsList();
+
+        // 'lts' is a special version name that refers to the current long term support version
+        if (self::VERSION_LTS === $this->version) {
+            if ($versionsList && isset($versionsList[$this->version])) {
+                // Get the latest patch of the lts version
+                $this->version = $versionsList[$this->version];
```

Comment:
There is a `lts` key in the `http://symfony.com/versions.json` file. As the `$this->version` contains `lts` when reaching this line, it retrieves the correct element.